include/vinecopulib/bicop/class.hpp

```diff
@@ -86,6 +86,9 @@ class Bicop
 
     double bic(const Eigen::Matrix<double, Eigen::Dynamic, 2> &u) const;
 
+    double mbic(const Eigen::Matrix<double, Eigen::Dynamic, 2> &u, 
+                const double pi) const;
```

Comment:
should be psi0

Reply:
or am I missing something?

Reply:
oh right, that's debatable. psi0 is really meant to be the "base prior" in the mBICV and "pi" just the prior for a single pair. but I agree that it's inconsistent

Reply:
yeah then I would just avoid pi, I don't really like to you pi for anything else than pi

Reply:
(Bayesians are weird)

Reply:
yeah, I'll change to psi0, got caught up too much in those weird Bayesian papers ;)